### PR TITLE
Add thanks button in the popup

### PR DESCRIPTION
**Pull request by Powersource**
Stuff left to fix:

- [x] Fix merge conflicts
- [x] Make the popup look slightly better (vuetify)
- [x] Show the number of times you've thanked the current page
- [x] Attach creators to the thanks

### CODE REVIEW

File: src/popup/popup.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+div(style="width: 300px; padding: 20px")
+  h1 Thankful
+
+  hr
```
Comment (ErikBjare) [inline src/popup/popup.vue]: Remove these fucking hr's, or I'll report you to HR.

File: src/lib/db.js [hunk 2]
```diff
@@ -230,7 +230,7 @@ export class Database {
       if (user_or_org.length > 0 && !isReserved.check(user_or_org)) {
         let creator_url = `https://github.com/${user_or_org}`;
         await new Creator(creator_url, user_or_org).save();
-        await this.connectActivityToCreator(a.url, creator_url);
+        await this.connectLogToCreator(a.url, creator_url);
```
Comment (ErikBjare) [inline src/lib/db.js]: This namechange makes no sense. It doesn't contain "Log" to creator, it gives an Activity object in the database a creator.

File: src/lib/db.js [hunk 3]
```diff
@@ -188,7 +188,7 @@ export class Database {
     });
   }
 
-  async connectActivityToCreator(url, creator) {
+  async connectLogToCreator(url, creator) {
```
Comment (ErikBjare) [inline src/lib/db.js]: Sorry here was the function that should be renamed (see other comment).

Should also be rewritten as to not depend on `this.logActivity`:

```
let activity = new Activity(url);  // Will handle canocalization
activity.creator = a.url;
return await activity.save();
```

File: src/lib/db.js (hunk 2 above)
Comment (Powersource) [reply src/lib/db.js]: Check new commits.

File: src/lib/db.js (hunk 3 above)
Comment (Powersource) [reply src/lib/db.js]: Should I do that in this PR?

Comment (ErikBjare) [reply src/lib/db.js]: @Powersource Yes.

File: src/lib/db.js [hunk 4]
```diff
@@ -72,6 +68,24 @@ export class Donation {
   }
 }
 
+export class Thank {
+  constructor(url, title, creator) {
+    this.url = canonicalizeUrl(url);
+    this.date = new Date();
```
[the remark this answers is not in the document]
Comment (Powersource) [reply src/lib/db.js]: And yeah I'll add a function that does both.

[the remark this answers is not in the document]
Comment (ErikBjare) [reply src/lib/db.js]: Yes, they all need to be in the UTC timezone (which `new Date().toISOString()` does).

**Pull request merged**